responsive-test-utils/src/test/java/dev/responsive/kafka/api/TopologyTestDriverExampleTest.java

```diff
@@ -31,19 +31,19 @@
 import org.apache.kafka.streams.kstream.KTable;
 import org.hamcrest.MatcherAssert;
 import org.hamcrest.Matchers;
+import org.junit.jupiter.api.Test;
 
 public class TopologyTestDriverExampleTest {
 
-  // TODO: re-enable this test once ResponsiveTopologyTestDriver is fully implemented
-  // @Test
+  @Test
   public void shouldRunWithoutResponsiveConnection() {
     // Given:
     final Properties props = new Properties();
     props.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.StringSerde.class);
     props.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.StringSerde.class);
 
-    final Topology topo = topology();
-    final TopologyTestDriver driver = new ResponsiveTopologyTestDriver(topo, props);
+    final Topology topology = topology();
+    final TopologyTestDriver driver = new ResponsiveTopologyTestDriver(topology, props);
```

Comment:
Well I personally know that it's using the Responsive stuff just from debugging the CassandraClient/Schema/Writer stubs lol -- but I know what you mean.

This might best be achieved by checking the return values once we have those stubbed instead of returning null everywhere as this PR currently does. Anyways I'm going to do the ttl + expanded test coverage in an (immediate) followup PR so I'll see what we can do there